x-pack/test_serverless/api_integration/test_suites/common/elasticsearch_api/home.ts

```diff
@@ -6,14 +6,28 @@
  */
 
 import { FtrProviderContext } from '../../../ftr_provider_context';
-
+import { RoleCredentials } from '../../../../shared/services';
 export default function ({ getService }: FtrProviderContext) {
-  const esSupertest = getService('esSupertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+
   const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  let roleAuthc: RoleCredentials;
 
   describe('Home', function () {
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+    });
+    after(async () => {
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
+    });
     it('can request /', async () => {
-      const { body, status } = await esSupertest.get('/');
+      const { body, status } = await supertestWithoutAuth
+        .get('/')
+        .set(svlCommonApi.getCommonRequestHeader())
+        .set(roleAuthc.apiKeyHeader)
+        .redirects(2);
```

Comment:
_question_: could you explain why do we need to allow redirects here?

Reply:
Honestly, I do not know.  
What I do know, is that the request fails without this, after I added my changes.

Reply:
Ok, let's double check with code owners here

Reply:
That'd be @pheyos :)